virt/virt-4-13-release-notes.adoc

```diff
@@ -78,6 +78,7 @@ The SVVP Certification applies to:
 === Storage
 
 //CNV-18462 Release note: CHANGE Retire alpha APIs for Storage
+//CNV-18462 Release note: CHANGE  - Retire Storage Alpha APIs
```

Comment:
These first two are duplicates. It otherwise LGTM.

Reply:
I removed one of the instances in the file.